### PR TITLE
Refresh website

### LINKED ISSUE #45: copyright year is out of date

### PR DESCRIPTION
**Pull request by geoknee**
To the reviewer: the commit messages do a decent job of explaining the changes. 

Also, bear in mind that the link to "the docs" will eventually point to https://statechannels.github.io/go-nitro/ (which is being tracked here https://github.com/statechannels/go-nitro/pull/872 )

closes #45

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,138 +1,320 @@
...
-  <meta property="og:image:width" content="400" />
-  <meta property="og:image:height" content="400" />
-  
-  <meta name="twitter:title" content="State Channels - Instant P2P Ethereum Transactions">
-  <meta name="twitter:description" content="State channels are a scalability technology for ethereum enabling instant & free peer-to-peer ethereum transactions">
-  <meta name="twitter:card" content="summary">
-  <meta name="twitter:url" content="https://statechannels.org/">
-  <meta name="twitter:image" content="https://statechannels.org/assets/symbol.png">
+  <head>
+    <meta charset="utf-8" />
```
Comment (kerzhner) [inline public/index.html:4]: It's unfortunate that this commit includes format changes as that makes the diff quite large.

**Review by kerzhner — APPROVED** [review]
Have we tested the new changes in a mobile browser?

File: public/index.html (hunk 1 above)
Comment (geoknee) [reply public/index.html:4]: Apologies. I've made another PR to add `prettier` so we don't get that on future PRs 🙏 .

Comment (geoknee) [reply public/index.html:4]: #53

**Pull request merged**